src/unread/__tests__/unreadMentionsReducer-test.js

```diff
@@ -54,13 +56,7 @@ describe('unreadMentionsReducer', () => {
     test('if message does not contain "mentioned" flag, do not mutate state', () => {
       const initialState = deepFreeze([]);
 
-      const action = deepFreeze({
-        type: EVENT_NEW_MESSAGE,
-        message: {
-          id: 2,
-          flags: [],
-        },
-      });
+      const action = mkMessageAction(eg.streamMessage({ flags: [] }));
```

Comment:
Nice! Glad to get more use out of Greg's `mkMessageAction`. Nice that it reduces the number of needed lines.